packages/data-service/src/data-service.ts

```diff
@@ -76,11 +76,20 @@ import type { ConnectionStatusWithPrivileges } from './run-command';
 import { runCommand } from './run-command';
 import type { CSFLECollectionTracker } from './csfle-collection-tracker';
 import { CSFLECollectionTrackerImpl } from './csfle-collection-tracker';
-import { ClientEncryption } from 'mongodb-client-encryption';
+
+import * as mongodb from 'mongodb';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore no types for 'extension' available
```

Comment:
I think we could if we wanted to, but practically speaking, it’s probably not worth the effort – I think it might just be something for when the entire addon is converted to TS (which is something that I hope will happen eventually :slightly_smiling_face:)